src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -837,6 +840,16 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 		return tabContainer;
 	}
 
+	private toEditorIndex(tabIndex: number): number {
```

Comment:
Fyi in a previous change I had inlined your `getEditorIndex` method because it was only used once, but now its used twice so I bring it back.